src/constants/storage.js

```diff
@@ -13,4 +13,5 @@ export const STORAGE_PROMETHEUS_QUERIES = {
   UTILIZATION_STORAGE_USED_QUERY: 'ceph_cluster_total_used_bytes[60m:5m]',
   UTILIZATION_STORAGE_USED_DEFAULT_QUERY:
     '(sum(node_filesystem_avail_bytes) - sum(node_filesystem_free_bytes))[60m:5m]',
+  UTILIZATION_STORAGE_IORW_QUERY: '(sum(rate(ceph_pool_wr_bytes[1m]) + rate(ceph_pool_rd_bytes[1m])))[60m:1m]',
```

Comment:
@cloudbehl , @gnehapk , @nthomas-redhat, can you please confirm this query properly shows the expected Disk IO RW utilization, please?